src/ochanticipy/datasources/codab/codab.py

```diff
@@ -147,38 +171,50 @@ def load_custom(self, custom_layer_number: int = 0) -> gpd.GeoDataFrame:
         try:
             # Ignore mypy for this line because custom_layer_names could be
             # None, but this is handled by the caught exceptions
-            layer_name = self._datasource_config.custom_layer_names[
+            custom_layer = self._datasource_config.custom_layer_names[
                 custom_layer_number
             ]  # type: ignore
+
+            layer_name = custom_layer.name
+            admin_level = custom_layer.hdx_resource
+
         except (IndexError, TypeError) as err:
             raise AttributeError(
                 f"{custom_layer_number}th custom layer requested but not "
                 f"available in {self._country_config.iso3.upper()} config file"
             ) from err
-        return self._load_admin_layer(layer_name=layer_name)
+        return self._load_admin_layer(
+            layer_name=layer_name, admin_level=admin_level
+        )
+
+    def _load_admin_layer(
+        self, layer_name: str, admin_level: Optional[int] = None
```

Comment:
I think we should just make admin level default 0 in the `load` method, I've been meaning to change that anyway for awhile